src/sentry/api/serializers/models/role.py

```diff
@@ -55,6 +55,7 @@ def serialize(
             "allowed": obj in allowed_roles,  # backward compatibility
             "isAllowed": obj in allowed_roles,
             "isRetired": is_retired_role,
+            "isTeamRolesAllowed": obj.is_team_roles_allowed,
```

Comment:
hmm I feel like this variable name is slightly deceptive. When I think of team roles, I'm more imagining the [team-level roles](https://docs.sentry.io/product/accounts/membership/#team-level-roles), rather than can with role, can the user be added to teams.
Maybe something along the lines of `canJoinTeam`?

Reply:
In practice they are pretty much the same. The role's attribute `is_team_roles_allowed` is referring to Team-Level Roles, and indicates whether or not a given role may actually possess a Team-Level Role. Rather than change the attr that already exists, I'm just passing it along as written to the API rather than simplifying it.

Rather, in the other PR I'm simplifying the message we communicate to the user, but I don't think the API needs to be obfuscated from the data model.

Reply:
oh ok makes sense, lgtm